src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/os/OsPlacesApiResponse.kt

```diff
@@ -1,5 +1,5 @@
 package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.os
 
 data class OsPlacesApiResponse(
-  val results: List<OsPlacesApiAddress>,
+  val results: List<OsPlacesApiAddress>?,
```

Comment:
Normally it returns an empty array for no results found but I found one case where it returned nothing strangely, this allows us to handle that case